.github/workflows/ci.yaml

```diff
@@ -735,6 +693,7 @@ jobs:
             --test-group-count 4 \
             --test-group=${{ matrix.group }} \
             --cov homeassistant \
+            --cov-report= \
```

Comment:
This removes the possibility to check the coverage of files added in a new PR. For example, if a configuration flow is added, we require 100% coverage. This allowed for peeking.

Reply:
It's very spammy though. I usually go to codecov to check coverage. And I always need to scroll a thousand lines to get back up to the stack trace when looking at a failed CI run.

I'm in favor of hiding the coverage report in the CI log

Reply:
If only Codecov would work as advertised 😬

Reply:
The coverage report is still printed during the job. When they are combined. [Link](https://github.com/home-assistant/core/pull/46696/checks?check_run_id=1919493987#step:7:13)

Reply:
That is fine I guess 👍